RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -179,9 +179,10 @@ std::shared_ptr<const BeamSpotObjects> OnlineBeamSpotESProducer::produce(const B
     return std::shared_ptr<const BeamSpotObjects>(best, edm::do_nothing_deleter());
   } else {
     return std::shared_ptr<const BeamSpotObjects>(&fakeBS_, edm::do_nothing_deleter());
-    edm::LogWarning("OnlineBeamSpotESProducer")
+    /*    edm::LogWarning("OnlineBeamSpotESProducer")
```

Comment:
Perhaps in this case one could revert the order of the return and LogWarning, so that the warning can still be issued if needed

Reply:
same reason as https://github.com/cms-sw/cmssw/pull/43030#discussion_r1361606548

Reply:
Also here I think this was just inverted by mistake